lib/l10n/app_de.arb

```diff
@@ -0,0 +1,445 @@
+{
+    "@@locale": "de",
+
+    "@_readme": {
+        "notes": [
+            "Alle Zeichenketten beginnen mit einem Großbuchstaben.",
+            "Gruppieren Sie Zeichenketten nach Kategorie, aber fügen Sie nicht zu einem Bereich hinzu, wenn sie in mehreren Bereichen verwendet werden können.",
+            "Führen Sie check_strings.py für die .arb Datei aus, um Probleme zu finden. Passen Sie @_lint_rules nach Sprache an wie nötig."
+        ],
+        "prefixes": {
+            "s_": "Ein einzelnes Wort oder wenige Wörter. Sollte kurz genug sein, um auf einer Schaltfläche oder einer Überschrift angezeigt zu werden.",
+            "l_": "Eine einzelne Zeile, kann umbgebrochen werden. Sollte nicht mehr als einen Satz umfassen und nicht mit einem Punkt enden.",
+            "p_": "Ein oder mehrere ganze Sätze mit allen Satzzeichen.",
+            "q_": "Eine Frage, die mit einem Fragezeichen endet."
+        }
+    },
+
+    "@_lint_rules": {
+        "s_max_words": 4,
+        "s_max_length": 32
+    },
+
+    "app_name": "Yubico Authenticator",
+
+    "s_save": "Speichern",
+    "s_cancel": "Abbrechen",
+    "s_close": "Schließen",
+    "s_delete": "Löschen",
+    "s_quit": "Beenden",
+    "s_unlock": "Entsperren",
+    "s_calculate": "Berechnen",
+    "s_label": "Label",
```

Comment:
On line 342 you use "die Beschriftung" for label. Could that be used here as well, instead of Label?